CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java

```diff
@@ -0,0 +1,29 @@
+package competition.subsystems.collector.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import competition.subsystems.collector.CollectorSubsystem;
+import competition.subsystems.hanger.HookSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class UpdateCollectorSensorsCommand extends BaseCommand {
```

Comment:
"update sensors" isn't particularly descriptive -- we should normalize on a phrase that we use for all of this type of command. I've been calling it "telemetry".